modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -144,36 +234,284 @@ function inope_policy_review_form($form, &$form_state, $node) {
 
 /**
  * The iNope Policy Review Form submit handler.
+ *
+ * This submit handler is broken up into 5 parts.
+ * PART 1 - Set variables and load necessary objects.
+ * PART 2 - Update existing entries if policy review id is a non empty string.
+ * PART 3 - Insert an approved response.
+ * PART 4 - Insert a changes requested response.
+ * PART 5 - Catch errors.
  */
 function inope_policy_review_form_submit($form, &$form_state) {
-  // Load objects from which their properties will be displayed.
+  // PART 1 - Load objects from which their properties will be displayed.
   $account = user_load($form_state['values']['inope_policy_review_hidden_uid']);
```

Comment:
Additionally you may just want to be sure that both of these hidden values are integers before you node and user load.